### PR TITLE
style: SDDSIP-2058 - Sequelize config file added

### PR DESCRIPTION
**Pull request by richard-mohammed-def**
https://eaflood.atlassian.net/browse/SDDSIP-2058?focusedCommentId=574947

### CODE REVIEW

File: packages/api/sequelize.config.cjs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  development: {
```
Comment (StuAA78) [inline packages/api/sequelize.config.cjs:2]: I just checked and all our environments have been configured with `NODE_ENV` set to `production`. So this key needs to be named `production` for it to work in the environments. However it doesn't strike me as ideal that we have dev and test environments set as `production`! So for now, can this object also include `development` and `test` (with the same values). Then we're covered if in future we change `NODE_ENV` to more suitable values.

File: packages/api/sequelize.config.cjs [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    username: process.env.POSTGRES_USER,
+    password: process.env.POSTGRES_PW,
+    database: process.env.POSTGRES_DB,
+    host: process.env.POSTGRES_HOST,
```
Comment (StuAA78) [inline packages/api/sequelize.config.cjs:6]: Can we also add `port: process.env.POSTGRES_PORT` to this. I can't imagine we would set it to anything other than the default port `5432` but we do set it in the environment so we ought to use it here as well.

**Review by StuAA78 — CHANGES_REQUESTED** [review]
Nice one, just a couple of small tweaks requested but hopefully this will fix it!

**Review by jonathangoulding — APPROVED** [review]

**Review by StuAA78 — APPROVED** [review]
Great stuff 👍

**Pull request merged**